services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/factory/HankeKayttajaFactory.kt

```diff
@@ -270,5 +271,23 @@ class HankeKayttajaFactory(
             roolit: List<ContactType> = emptyList()
         ): List<HankeKayttajaDto> =
             (1..amount).map { createDto(it, roolit = roolit, tunnistautunut = (it % 2 == 0)) }
+
+        private fun createDto(
+            i: Int,
+            id: UUID = UUID.randomUUID(),
+            roolit: List<ContactType> = emptyList(),
+            tunnistautunut: Boolean = false
+        ) =
+            HankeKayttajaDto(
```

Comment:
I wonder if it would be better if createDto(id, etunimi, sukunimi, sahkoposti, puhelinnumero, kayttooikeustaso, roolit, tunnistautunut) would be called here instead of constructing HankeKayttajaDto in very similar manner as in that other createDto function?

Reply:
I changed this to use `createDto` instead.